### DISCUSSION TITLE
Webstorm 2023 Autocompletion

### DISCUSSION THREAD

**Discussion by david-rohrschneider** [tailwindlabs/tailwindcss#11010] [Help]
Hello, i am using tailwind inside of a vite/vue 3 project and everything works fine, but Webstorm doesnt show any auto completion for the classes. I have the newst webstorm version and the tailwind plugin is also installed.

**Comment by cnsource** [comment]
I've been experiencing this issue for a while too. Have you found a solution? In my case, I resolved it temporarily by changing the PostCSS configuration file to use CommonJS module format and using module.export. The same applies for the Tailwind configuration file. I believe the issue is caused by incompatibility between the built-in Tailwind plugin in JetBrains and the configuration file generated by Tailwind.

**Comment by dan-in-the-mirror** [comment]
Hey there David, cnsource and random readers,

I had the same issue ( three years later - but still april :) ). Just add an empty tailwind.config.mjs file to project root: ChaChing. Now the autocompletion works fine.

Have a nice day

dan-in-the-mirror

Source:
https://github.com/tailwindlabs/tailwindcss-intellisense/issues/1153 | viewed on 2025-04-27 14:45
